src/gui_common/fonts/Lato-BoldItalic-Small.tres

```diff
@@ -0,0 +1,26 @@
+[gd_resource type="DynamicFont" load_steps=11 format=2]
+
+[ext_resource path="res://assets/fonts/NotoSansSinhala-Regular.ttf" type="DynamicFontData" id=1]
```

Comment:
Could you reorder these font files to have the regular font item first, so in this case the one with id 10 should be the one with id 1? So that this is consistent with the other font files. This is required because I use multi file editing to not lose my mind when I need to add new fallback fonts.

Reply:
I just duplicated a dynamic font file, forgot fallback fonts exist, will clean this up.